documentation/release_3.1.htm

```diff
@@ -161,7 +162,8 @@ <h3>Changed functionality</h3>
 </li>
 <li>
 The orientation of images read/written via ITK has changed. It should now be correct if the patient
-was in HFS position. This is currently not checked (and for many file formats, never can be checked).
+was in HFS position for NIfTI (this can't be verified due to lack of metadata in the format), or if
```

Comment:
I find this a bit awkward to read. I suggest 

> was in HFS position. Unfortunately, for most file formats (such as NifTI) this can't be verified due to lack of metadata in the format. If the files are in DICOM format, as long as the file contains the correct metadata, other patient positions are supported (currently HFS, FFS, HFP or FFP).
```

Reply:
Yes was a bit muddled in my head as I tried to get the words out. Much better